dist/iidx/iidxhook-27.conf

```diff
@@ -22,6 +22,9 @@ io.tt_multiplier=1.0
 # Disables the camera emulation
 cam.disable_emu=true
 
+# Camera port layout (0 = LDJ, 1 = CLDJ/TDJ-JA, 2 = TDJ-JB)
```

Comment:
Can you explain what the port layout implies here regarding which one to chose, e.g. in a home setup? Might be something for the readme.